front/src/modules/simulationResult/components/ManchetteWithSpaceTimeChart/ManchetteWithSpaceTimeChart.tsx

```diff
@@ -42,7 +48,45 @@ const ManchetteWithSpaceTimeChartWrapper = ({
   );
 
   const [showSettingsPanel, setShowSettingsPanel] = useState(false);
-  const [settings, setSettings] = useState({ showConflicts: false });
+  const [settings, setSettings] = useState({
+    showConflicts: false,
+    showBAL: false,
+    showBAPR: false,
+    showTVM: false,
+    showSignalsStates: false,
+  });
+
+  const occupancyBlocks = projectPathTrainResult
+    .filter((train) => train.signal_updates)
```

Comment:
I wonder, do we need this `filter()`? I think the map below should handle empty arrays just fine?

Reply:
No you're right, fixed